documentation/modules/deploying/proc-deploy-user-operator-standalone.adoc

```diff
@@ -133,6 +135,9 @@ This helps to avoid unnecessary exceptions in the Kafka cluster logs.
 The default is `true`.
 <18> (Optional) Semi-colon separated list of Cron Expressions defining the maintenance time windows during which the expiring user certificates will be renewed.
 <19> (Optional) Configuration options for configuring the Kafka Admin client used by the User Operator in the properties format.
+<20> (Optional) Indicates whether the Kafka cluster the User Operator is connecting to is using ZooKeeper or KRaft.
+When running against KRaft clusters, some features such as management of SCRAM-SHA-512 users is disabled because Apache Kafka does currently not support it.
+The default is `false`.
```

Comment:
```suggestion
<20> (Optional) Indicates whether the Kafka cluster the User Operator is connecting is using KRaft instead of ZooKeeper.
Set this variable to `true` if the Kafka cluster uses KRaft. The default is `false`.
Note that some features are not available when running against KRaft clusters. 
For example, management of SCRAM-SHA-512 users is disabled because Apache Kafka currently does not support it.
```